roles/installer/molecule/satellite/converge.yml

```diff
@@ -1,12 +1,12 @@
 ---
 - name: Converge
   hosts: all
-  gather_facts: false
+  gather_facts: true
   vars:
-    foreman_installer_scenario: foreman
+    satellite_installer_scenario: foreman
```

Comment:
I could try to fake through symlinks a satellite scenario? That felt a bit odd (granted I just started creating an RPM so...)

Reply:
Given we need to pass the scenario anyways, I don't see a huge benefit (vs the package where we want to test that it actually does the right thing by default).